ocaml/xcp-rrdd/bin/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -332,7 +332,8 @@ let refresh_phypath_to_sr_vdi () =
 let exec_tap_ctl_list () : ((string * string) * int) list =
   let tap_ctl = "/usr/sbin/tap-ctl list" in
   let extract_vdis pid minor _state kind phypath =
-    if not (kind = "vhd" || kind = "aio") then raise (Failure "Unknown type") ;
+    if not (kind = "vhd" || kind = "aio" || kind = "qcow2") then
+      raise (Failure "Unknown type") ;
```

Comment:
Would be good to report the actual type that was found.

Reply:
yes sure. You mean something like: `raise (Failure (Printf.sprintf "%s: unknown type" kind)) ;`
Note that currently when an wrong type is detected the line is logged as the "Offending line": 
```
Apr 24 17:41:13 xcp-gtn-ip13 xcp-rrdd-iostat: [ warn||0 ||xcp-rrdd-iostat] "/usr/sbin/tap-ctl list" returned a line that could not be parsed. Ignoring.
Apr 24 17:41:13 xcp-gtn-ip13 xcp-rrdd-iostat: [ warn||0 ||xcp-rrdd-iostat] Offending line: pid=3886 minor=0 state=0 args=qcow2:/var/run/sr-mount/d835d019-1b97-0080-1bc0-eb52542db3f5/ce067f8f-0851-4206-9536-191b
01c39dcb.qcow2
```

Reply:
And in fact I just notice when pasting the log that it looks like the Failure string is ignored currently and replaced by the offending line... Oh yes I checked so I probably need to modify `read_out_line`. But yes it is not difficult and it will help to find issue.

Reply:
Or add D.warn in fact. I will do that.